mozci/task.py

```diff
@@ -123,6 +124,19 @@ def is_bad_group(task_id: str, group: str) -> bool:
     return bad_group
 
 
+def is_autoclassifiable(task: Task) -> bool:
+    """Check a task is enabled for auto-classification
+    by applying glob patterns from configuration
+    """
+    if not task.label or not config["autoclassification"]["enabled"]:
```

Comment:
Maybe we should assert that `task.label` is present

Reply:
Done